http_check/check.py

```diff
@@ -319,6 +320,14 @@ def send_status_down(loginfo, message):
             else:
                 send_status_up("%s is UP" % addr)
 
+        # Report status as a metric as well
+        can_status = 1 if service_checks[0][1] == "UP" else 0
```

Comment:
Service checks are sent in a different way than the metrics. I think it's best not to rely upon the `service_checks` object to always exist in the same way, and instead to set them independently of them. That's generally how we do it elsewhere and I think that's how we should do it here as well.